protocol/communities/manager.go

```diff
@@ -2119,25 +2119,17 @@ func (m *Manager) DeclineRequestToJoin(dbRequest *RequestToJoin) (*Community, er
 	return community, nil
 }
 
-func (m *Manager) isUserRejectedFromCommunity(signer *ecdsa.PublicKey, community *Community, requestClock uint64) (bool, error) {
-	declinedRequestsToJoin, err := m.persistence.DeclinedRequestsToJoinForCommunity(community.ID())
+func (m *Manager) shouldUserRetainDeclined(signer *ecdsa.PublicKey, community *Community, requestClock uint64) (bool, error) {
+	requestID := CalculateRequestID(common.PubkeyToHex(signer), types.HexBytes(community.IDString()))
+	request, err := m.persistence.GetRequestToJoin(requestID)
```

Comment:
What if we request a declined request directly? As I understand, if it is not declined - it will return `false, nil` anyway (the same as empty)

Reply:
Not sure I understand, could you please elaborate?